src/devices/Sgp30/README.md

```diff
@@ -0,0 +1,45 @@
+﻿# Sensorion SGP30 TVOC + eCO2 Gas Sensor
+This is a device binding for the Sensorion SGP30 TVOC + eCO2 Gas Sensor.
+
+## Datasheet
+[SGP30](https://cdn.shopify.com/s/files/1/0174/1800/files/Sensirion_Gas_Sensors_SGP30_Datasheet_EN-1148053.pdf)
+
+## Binding Notes
+**NOTE**: Once initialised, this sensor should be queried at least once per second. If more than several seconds pass,
+the initialisation routine should be called again to ensure the device is returning valid data.
+
+```cs
+// Setup Device Instance
+I2cDevice sgp30Device = I2cDevice.Create(new I2cConnectionSettings(1, Sgp30.DefaultI2cAddress));
+Sgp30 sgp30 = new Sgp30(sgp30Device);
+
+// Initialise the SGP30 device
+Console.WriteLine("Wait for initialisation (upto 20 seconds).");
```

Comment:
Can you mention that the wait is done in the sensor code? so it's clear that the developer does not have to wait themselves.